Community feedback, move to draft-iab

britram opened the pull request.

britram committed:
create draft-iab-00

britram committed:
community feedback wip

tfpauly commented on draft-iab-wire-image.md:
```diff
@@ -61,53 +61,69 @@ discuss in this document. It is the wire image, not the protocol's
+of its own control information, bring new relevance to this question. QUIC is,
+in effect, the first IETF-defined transport protocol to take care of the
+minimization of its own wire image, for reasons of ossification prevention as
+well as reducing information radiation off the protocol for end-to-end privacy.
```
This last sentence seems a bit wordy.

Perhaps:
`QUIC is, in effect, the first IETF-defined transport protocol to take care of the minimization of its own wire image, to prevent ossification and improve end-to-end privacy by reducing information radiation.`

britram replied:
yep done, thx.

tfpauly commented on draft-iab-wire-image.md:
```diff
@@ -61,53 +61,69 @@ discuss in this document. It is the wire image, not the protocol's
-messages sent by each participant in the protocol, each expressed as a
-sequence of bits with an associated arbitrary-precision time at which it was
-sent.
+More formally, the wire image of a protocol stack observed at a given point
```
`Wire image of a protocol stack`? Or the `wire images of a set of protocols`?

britram replied:
I'm still trying out this language; it's to address Stephen's point that you can't consider any one protocol in isolation with respect to what it exposes, you have to consider all of the protocols above and below it... I'm leaning toward "stack" here but I might need to be a little more wordy about what exactly I mean -- it's not the software on the endhost, it's the... erm, as you say, set of protocols in use at a given point.

Will continue considering.

britram replied:
ah actually, yes, it's *set* of protocols because of associated interactions (control and data, DNS, etc.)

tfpauly commented on draft-iab-wire-image.md:
```diff
@@ -61,53 +61,69 @@ discuss in this document. It is the wire image, not the protocol's
-sent.
+More formally, the wire image of a protocol stack observed at a given point
+in the network consists of the sequence of packets sent by each participant in
+the communication, each expressed as a sequence of bits with an associated
```
Nit: I think "with *the* associated... time at which it", not "with *an* associated", flows better for discussing a specific "each" packet.

britram replied:
yep thx

tfpauly commented on draft-iab-wire-image.md:
```diff
@@ -61,53 +61,69 @@ discuss in this document. It is the wire image, not the protocol's
+For a given packet observed at a given point in the network, the wire image
+contains information from the entire stack of protocols in use at that
+observation point. Confidentiality and integrity protection may be added at
+multiple layers in the stack. However, PDUs at the transport layer and above are
```
Perhaps define PDU here, since it isn't used previously?

britram replied:
Or not use it at all; thanks for flagging...

tfpauly reviewed (commented):
Looks good in general!

britram commented:
thanks for the review @tfpauly! Not *quite* yet done incorporating community feedback from the list, once I am (and I've addressed your points here) can i ping you to have another look?

britram committed:
finish first pass community review and @tfpauly comments
